### PR TITLE
Modified route.js with two new functions

### PR DESCRIPTION
**Pull request by veerleprins**
Modified route.js with two new functions to filter on gender and movies. I also added an out commented line on the function 'home' to make my two new functions work. The first variable in this out commented line needs to be users from Jordy's functions and second from the user's ID that is logged in.

### CODE REVIEW

File: routes/route.js [hunk 1]
```diff
@@ -131,6 +131,7 @@ async function home(req, res, next) {
   // Routes function home, graps every user with 'seen: false' and shows them on page.
   try {
     let allUsers = await usersCollection.find({ seen: false }).toArray();
+    // let filtered = await checkGenderPref(hierin moet een array komen van gebruikers VOOR filteren, thisUser);
```
Comment (joordy) [inline routes/route.js:134]: So the users where I'm looking for (Not seen yet), should be there?

Comment (veerleprins) [reply routes/route.js:134]: Yess! and of course the ID of the user that's logged in

File: routes/route.js [hunk 2]
```diff
@@ -183,6 +184,34 @@ async function matchList(req, res, next) {
...
+  //a boolean if the conditions are correct for both sides:
+  if (loggedIn[0].prefMovies === "") {
+    return true;
+  } else if (loggedIn[0].prefMovies !== "") {
+    return user.movies.find(function (movie) {
+      return movie === loggedIn[0].prefMovies;
+    });
+  }
+}
+
```
Comment (joordy) [inline routes/route.js:214]: Nice! Good solution for so many code, keeps it nice and clean! 😍

Comment (veerleprins) [reply routes/route.js:214]: Thanks, I know right 😄 ?

**Comment by veerleprins** [comment]
@rowanhorn1412 Yes, except for a few minor adjustments! 😄

**Pull request merged**